systemvm/agent/scripts/ipfirewall.sh

```diff
@@ -32,7 +32,7 @@ config_htaccess() {
 }
 
 ips(){
-  echo "allow from $1" >> $HTACCESS
+  grep "^allow from $1$" $HTACCESS || echo "allow from $1" >> $HTACCESS
```

Comment:
to be sure 
```suggestion
  grep -e "^allow from $1$" $HTACCESS || echo "allow from $1" >> $HTACCESS
```
??

Reply:
@DaanHoogland 
thanks, committed

(maybe unnecessary)